Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
```

Comment:
How about a reply to a reply?

---
In reply to: [53665025](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r53665025) [](ancestor path = 53665025, 53665023)